tests/unittests/unittest.cpp

```diff
@@ -53,9 +89,10 @@
 #include "datamodel/MutableExampleWithComponent.h"
 #include "datamodel/MutableExampleWithExternalExtraCode.h"
 #include "datamodel/StructWithExtraCode.h"
-
 #include "podio/UserDataCollection.h"
 
+class ExampleWithOneRelationData;
+
```

Comment:
We include this already so we should be able to drop the forward declaration.

Reply:
Done